docs/manual/core-concepts/getting-started.md

```diff
@@ -7,18 +7,29 @@ In this tutorial you will learn to make a simple setup of Sequelize.
 Sequelize is available via [npm](https://www.npmjs.com/package/sequelize) (or [yarn](https://yarnpkg.com/package/sequelize)).
 
 ```sh
-npm install --save sequelize
+# using npm
+npm i sequelize # This will install latest version of Sequelize
+# using yarn
+yarn add sequelize
 ```
 
 You'll also have to manually install the driver for your database of choice:
 
 ```sh
-# One of the following:
-$ npm install --save pg pg-hstore # Postgres
-$ npm install --save mysql2
-$ npm install --save mariadb
-$ npm install --save sqlite3
-$ npm install --save tedious # Microsoft SQL Server
+# using npm
+$ npm i pg pg-hstore # PostgreSQL
```

Comment:
while we're changing these lines, can we remove the `$` character, like in the readme? (makes copy-paste harder)

Reply:
I wasn't sure why they were here so I left them. But I will remove these